src/content/inject.ts

```diff
@@ -24,11 +24,24 @@ window.onload = (): void => {
         }
       );
     },
+    (_: EventTarget) => {
+      chrome.runtime.sendMessage(
```

Comment:
If the user clicks on something that's not decryptable, the page is effectively unusable.

Reply:
Thanks for catching this!